lib/plugins/inspec-suggest/lib/inspec-suggest/suggtool/base.rb

```diff
@@ -55,6 +55,14 @@ def with_matching_files(set_cfg, &block)
         end
       end
 
+      def with_matching_yaml_files(set_cfg, &block)
```

Comment:
I would suggest renaming this to `with_set_metadata_files`

Reply:
Renamed!